Products/ZenHub/zenhubworker.py

```diff
@@ -299,6 +300,12 @@ def buildOptions(self):
             '--hubpassword', dest='hubpassword', default='zenoss',
             help="password to use when connecting to ZenHub",
         )
+        self.parser.add_option(
+            '--hub-connect-timeout', dest='hub_connect_timeout',
+            default=300, type='int',
```

Comment:
maybe 300 seconds is too long for a default.  it can be ramped up by those that need it.  Maybe 30 secs to start, as it was previously 10 secs?